framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2136,14 +2136,23 @@ protected void checkTypeArguments(Tree toptree,
     }
     */
 
-    // Overridden in the Lock Checker. This method is needed so that the
-    // Lock Checker does not need to override the checkMethodInvocability
-    // method as a whole. The Lock Checker needs the functionality in
-    // checkMethodInvocability, such as the receiver subtype checks.
+    /**
+     * Indicates whether to skip subtype checks on the receiver when
+     * checking method invocability. A visitor may, for example,
+     * allow a method to be invoked even if the receivers are siblings
+     * in a hierarchy, provided that some other condition (implemented
```

Comment:
Based on the method name, I thought this method was only used to skip the subtype check.  This comment implies that it is used to apply alternative constraints on a method invocation.  I think that's okay, but maybe remove the example in this comment?